docs/testnet/deploying-a-smart-contract.md

```diff
@@ -2,23 +2,35 @@
 Using the steps below you will add an extension to your MetaMask wallet so it can connect to Obscuro Testnet, then using Remix you will deploy your smart contract to Testnet.
 
 ## Prerequisites
-* MetaMask installed in your browser.
-* A local copy of the Obscuro MetaMask wallet extension downloaded from the [Obscuro repo on GitHub](https://github.com/obscuronet/go-obscuro/tree/main/tools/walletextension).
+* [MetaMask](https://metamask.io/) wallet installed in your browser.
+* A local copy of the Obscuro MetaMask wallet extension downloaded from the [releases area](https://github.com/obscuronet/go-obscuro/releases) Obscuro repository on GitHub.
 
 ## Prepare Your MetaMask Wallet for Obscuro Testnet
 An essential part of how Obscuro provides full privacy is the encryption of communication between an Obscuro application and Obscuro nodes on the network.
 
-Follow the steps to config the MetaMask wallet extension [here](https://docs.obscu.ro/testnet/wallet-extension.html) then return to this page.
+Follow the steps to config the MetaMask wallet extension [here](wallet-extension.md) then return to this page. If you do not have the Obscuro wallet extension running MetaMask will not be able to communicate with the Obscuro Testnet. 
```

Comment:
You need to both configure and run it (or you could add running it to the steps below).

Reply:
ok